include/onnxruntime/core/graph/graph.h

```diff
@@ -586,7 +595,7 @@ class Node {
   // create a Graph instance for an attribute that contains a GraphProto
   void CreateSubgraph(const std::string& attr_name);
 
-  const std::vector<std::unique_ptr<Graph>>& MutableSubgraphs() noexcept { return subgraphs_; }
+  std::vector<std::unique_ptr<Graph>>& MutableSubgraphs() noexcept { return subgraphs_; }
```

Comment:
nit: inconsistent naming. Some of the get functions are named like GetFoo(); some are just Foo().